# HG changeset patch
# User Rik <rik@octave.org>
# Date 1515093792 28800
#      Thu Jan 04 11:23:12 2018 -0800
# Branch stable
# Node ID 59d080a90f1760a60b5ba8ed6548137bc307359c
# Parent  4f0e6ee6c9b8455d528beb6fd70d826610964fab
doc: Remove unimplemented option "extra" from cumsum documentation.

* data.cc (Fcumsum): Remove @deftypefnx with "extra" option.
Remove mention of "extra" in docstring.

diff --git a/libinterp/corefcn/data.cc b/libinterp/corefcn/data.cc
--- a/libinterp/corefcn/data.cc
+++ b/libinterp/corefcn/data.cc
@@ -1171,33 +1171,32 @@ cumprod ([1, 2; 3, 4; 5, 6])
 */
 
 DEFUN (cumsum, args, ,
        doc: /* -*- texinfo -*-
 @deftypefn  {} {} cumsum (@var{x})
 @deftypefnx {} {} cumsum (@var{x}, @var{dim})
 @deftypefnx {} {} cumsum (@dots{}, "native")
 @deftypefnx {} {} cumsum (@dots{}, "double")
-@deftypefnx {} {} cumsum (@dots{}, "extra")
 Cumulative sum of elements along dimension @var{dim}.
 
 If @var{dim} is omitted, it defaults to the first non-singleton dimension.
 For example:
 
 @example
 @group
 cumsum ([1, 2; 3, 4; 5, 6])
    @result{}  1   2
        4   6
        9  12
 @end group
 @end example
 
-See @code{sum} for an explanation of the optional parameters
-@qcode{"native"}, @qcode{"double"}, and @qcode{"extra"}.
+See @code{sum} for an explanation of the optional parameters @qcode{"native"}
+and @qcode{"double"}.
 @seealso{sum, cumprod}
 @end deftypefn */)
 {
   int nargin = args.length ();
 
   bool isnative = false;
   bool isdouble = false;
 
